README.md

```diff
@@ -192,6 +192,7 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [Iguazio](https://www.iguazio.com/) - Data science platform that automates MLOps with end-to-end machine learning pipelines.
 * [Knime](https://www.knime.com/) - Create and productionize data science using one easy and intuitive environment.
 * [Kubeflow](https://www.kubeflow.org/) - Making deployments of ML workflows on Kubernetes simple, portable and scalable.
+* [Katonic.ai](https://katonic.ai/) - Automate your cycle of Intelligence with Katonic MLOps Platform.
```

Comment:
Probably Katonic should come before Knime...

Reply:
Okay I'll Update Sorry